main.tf

```diff
@@ -1,7 +1,7 @@
 resource "aws_security_group" "bastion" {
   name        = var.name
   vpc_id      = var.vpc_id
-  description = "Bastion security group (only SSH inbound access is allowed)"
+  description = "${var.aws_security_group_description}"
```

Comment:
Please update to use Terraform 0.12 syntax:

```
description = var.aws_security_group_description
```

Reply:
Done